masterlist.yaml

```
@@ -6843,6 +6843,41 @@ plugins:
         util: '[TES4Edit v4.0.3g](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 7
 
+  - name: 'Bounty Quests.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/47425/'
+        name: 'Bounty Quests'
+      - link: 'https://www.nexusmods.com/oblivion/mods/48330/'
+        name: 'Bounty Quests Fixed and Polished'
+    req:
+      - *OBSE
+      - 'DLCShiveringIsles.esp'
+    msg:
+      - <<: *useVersion
+        subs: [ '[Bounty Quests Fixed and Polished](https://www.nexusmods.com/oblivion/mods/48330/)' ]
+        condition: 'active("Bounty Quests OOO Patch.esp") and checksum("Bounty Quests.esp", F62CD624)'
+      - <<: *patchProvided
+        subs: [ 'Oscuro''s Oblivion Overhaul' ]
+        condition: 'active("Oscuro''s_Oblivion_Overhaul.esm") and not active("Bounty Quests OOO Patch.esp") and file("ini/Bounty Quests (Vanilla|OOO)\.ini")'
+      - <<: *patchUpdateAvailable
+        subs: [ '[Bounty Quests Fixed and Polished](https://www.nexusmods.com/oblivion/mods/48330/)' ]
+        condition: 'not file("ini/Bounty Quests (Vanilla|OOO)\.ini")'
```

The messages just need to be sorted.

```suggestion
      - <<: *patchProvided
        subs: [ 'Oscuro''s Oblivion Overhaul' ]
        condition: 'active("Oscuro''s_Oblivion_Overhaul.esm") and not active("Bounty Quests OOO Patch.esp") and file("ini/Bounty Quests (Vanilla|OOO)\.ini")'
      - <<: *patchUpdateAvailable
        subs: [ '[Bounty Quests Fixed and Polished](https://www.nexusmods.com/oblivion/mods/48330/)' ]
        condition: 'not file("ini/Bounty Quests (Vanilla|OOO)\.ini")'
      - <<: *useVersion
        subs: [ '[Bounty Quests Fixed and Polished](https://www.nexusmods.com/oblivion/mods/48330/)' ]
        condition: 'active("Bounty Quests OOO Patch.esp") and checksum("Bounty Quests.esp", F62CD624)'
```